Домашнее задание 5/Домашнее задание 5/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Домашнее_задание_5
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            double a = Program.Formula(2, 5);
+            double b = Program.Formula(3, 7);
+            double c = Program.Formula(5,11);
+
+            double x = a + b + c;
+
+            Console.WriteLine(Math.Round(x, 2));
+
+        }
+        static double Formula(double y, double z)
+        {
+            return (y + Math.Sin(y)) / (z + Math.Cos(z));
+        }
```

Comment:
А вот методы в классе лучше разделять пустой строкой. Так код воспринимается лучше.
Как уже было сказано выше, метод с двумя параметрами не соответствует заданию.
```suggestion
        }
        
        static double Formula(double y, double z, double w)
        {
            return (y + Math.Sin(y)) / (z + Math.Cos(w));
        }
```